tests/integration/targets/ios_vrf_global/tests/cli/_parsed.cfg

```diff
@@ -0,0 +1,11 @@
+vrf definition test
+ vnet tag 34
+ description This is test VRF
+ ipv4 multicast multitopology
+ ipv6 multicast multitopology
+ rd 10.2.3.4:300
```

Comment:
You can make use of [this](https://en.wikipedia.org/wiki/Reserved_IP_addresses) to find out documentation IP address ranges for examples/ tests.